src/organizations/selectors/index.ts

```diff
@@ -18,6 +18,12 @@ export const isOrgIOx = (state: AppState): boolean => {
   )
 }
 
+export const selectOrgCreationDate = (state: AppState): string => {
+  const org = getOrg(state)
+
+  return org.createdAt
+}
+
```

Comment:
More of an FYI than anything: This is `createdAt` from the idpe org endpoint (stored in redux state in resources). In a perfect world we'd use the `creationDate` property from quartz (stored in redux state under identity). 

Can't easily do that here though, since it would require changing how we make these API calls (`creationDate` only comes back from /orgs/:orgId, which only gets called and consumed by the orgs settings page). This can get handled in other tickets if the 'remove unnecessary idpe calls' work gets prioritized again at some future date.